docs/data/sdks/flutter/index.md

```diff
@@ -89,12 +89,12 @@ To support high performance environments, the SDK sends events in batches. Every
 
 ```dart
 // Events queued in memory will flush when number of events exceed upload threshold
-// Default value is 10
-Amplitude.getInstance().setEventUploadThreshold(20);
+// Default value is 30
+Amplitude.getInstance().setEventUploadThreshold(40);
 
 // Events queue will flush every certain milliseconds based on setting
-// Default value is 10,000 milliseconds
-Amplitude.getInstance().setEventUploadPeriodMillis(20000);
+// Default value is 30,000 milliseconds
+Amplitude.getInstance().setEventUploadPeriodMillis(40000);
```

Comment:
nit: trying to think of the best example values to use here. Would lower values make more sense for an example?

```
// Default value is 30
Amplitude.getInstance().setEventUploadThreshold(1);

// Default value is 30,000 milliseconds
Amplitude.getInstance().setEventUploadPeriodMillis(1000);
```